### ISSUE TITLE
KeyError: 'LANGUAGE_CODE'

### ISSUE THREAD

**Issue by saschalalala** [zostera/django-bootstrap4#52]
While debugging a template that derives from bootstrap4.html, I noticed that the html lang attribute is not set even if I set it in my settings.py.

I'm not really sure, where this LANGUAGE_CODE should come from without loading i18n before, so I changed the bootstrap4.html like this:

```html
<!DOCTYPE html>
{% load bootstrap4 %}
{% load i18n %}
{% get_current_language as LANGUAGE_CODE %}
<html{% if LANGUAGE_CODE %} lang="{{ LANGUAGE_CODE }}"{% endif %}>
```

which works for me. Perhaps this could be further optimized to check if i18n is enabled but I'm not sure how to do this at the moment.

**Comment by FlipperPA** [comment]
Could we do this instead, to set `en_us` as a default?

```html
<html lang="{{ LANGUAGE_CODE|default:'en_us' }}">
```

**Comment by dyve** [comment]
Sure, or leave it out if LANGUAGE_CODEI actually like @sachalalala's solution, but it might require django.i18n in settings.

**Comment by saschalalala** [comment]
I could try to find out if this can somehow be checked inside the template. `{% load i18n %}` will only work if `USE_I18N = True` in the settings, right? Then this could be checked inside bootstrap4.py and loaded inside the template depending on the setting. I could prepare a pull request this afternoon or tomorrow.

**Comment by dyve** [comment]
Alternative would be to just leave it out. I like to specify the language, but it is not required. Maybe we're trying to do too much, it's not a Bootstrap thing

**Comment by saschalalala** [comment]
I was thinking of it as an optional base theme for an application and as such I would expect it to be able to set the language from my settings inside the html lang attribute. In my current application I use it like so inside my base template:

`{% extends 'bootstrap4/bootstrap4.html' %}`

The thing is: The fix is quite small (About 5-6 lines in bootstrap4.py, 3-4 lines inside bootstrap4.html) and wouldn't affect users without i18n at all.

**Comment by jieter** [comment]
Fixed by #53
